app/src/main/java/woowacourse/movie/domain/ReservationDetail.kt

```diff
@@ -0,0 +1,9 @@
+package woowacourse.movie.domain
+
+import java.io.Serializable
+import java.time.LocalDateTime
+
+data class ReservationDetail(val date: LocalDateTime, val peopleCount: Int, val price: Price) :
+    Serializable {
+    fun getTotalPrice(): Int = peopleCount * price.value
```

Comment:
getter함수보다는 좀더 코틀린스럽게 사용해보아요! 
```suggestion
    val totalPrice: Int
        get() = peopleCount * price.value
```